src/index.scss

```
@@ -1,8 +1,8 @@
 /*!
- * Primer
- * http://primer.github.io
+ * Primer CSS
+ * https://primer.style
  *
- * Released under MIT license. Copyright (c) 2019 GitHub Inc.
+ * Released under MIT license. Copyright (c) 2020 GitHub Inc.
```

Maybe we can just remove the copyright altogether. So that we don't need to update it in 6 months again. 😉 

```suggestion
 * Released under MIT license.
```